firmware/throttle/kia_soul_ps/throttle_control_module.ino

```diff
@@ -495,52 +596,48 @@ void loop()
 {
 
     // update the global system update timestamp, ms
-    last_update_ms = GET_TIMESTAMP_MS();
+    last_update_ms = GET_TIMESTAMP_MS( );
 
-    handle_ready_rx_frames();
+    // checks for CAN frames, if yes, updates state variables
+    handle_ready_rx_frames( );
 
-    publish_timed_tx_frames();
+    // publish all report CAN frames
+    publish_timed_tx_frames( );
 
-    check_rx_timeouts();
+    // heartbeat checker??
+    check_rx_timeouts( );
 
     // update state variables
-    PSensL_current = analogRead(SIGNAL_INPUT_A) << 2;  //10 bit to 12 bit
-    PSensH_current = analogRead(SIGNAL_INPUT_B) << 2;
-    
+    signal_L_current = analogRead( SIGNAL_INPUT_A ) << 2;  //10 bit to 12 bit
+    signal_H_current = analogRead( SIGNAL_INPUT_B ) << 2;
+
     // if someone is pressing the throttle pedal disable control
-    if ( ( PSensL_current + PSensH_current) / 2 > PEDAL_THRESH ) {
-        disableControl();
-        local_override = 1;
+    check_pedal_override( );
 
-    } 
-    else 
+    // read and parse incoming serial commands
+    if (Serial.available() > 0)
     {
-        local_override = 0;
+        incoming_serial_byte = Serial.read( );
+        process_serial_byte( );
     }
 
-    // read and parse incoming serial commands
-    if (Serial.available() > 0) {
-        incomingSerialByte = Serial.read();
-        processSerialByte();
+    if ( ++loop_counter % 10 == 0 )
```

Comment:
Could make this into a macro to be cleaner (and more descriptive), but being picky here really a suggestion.